prow/config/config.go

```diff
@@ -1919,6 +1940,29 @@ func ValidateController(c *Controller) error {
 	return nil
 }
 
+func ValidateReportTemplates(c *Controller) error {
```

Comment:
I don't think this needs to be exporters. Also, can we rename to `DefaultAndValidateReportTemplate`? I wouldn't expect a fund called `ValidateReportTemplates` to mutate, but that's what we do